zilliqa/src/message.rs

```diff
@@ -125,11 +133,11 @@ pub struct BlockBatchRequest(pub BlockRef);
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct BlockResponse {
-    pub block: Block,
+    pub proposal: Proposal,
```

Comment:
What's the reason for this change? If I understand correctly, this will now mean blocks and the inner transactions will all be included in a response?

Reply:
yes, see my comment above:

<img width="954" alt="image" src="https://github.com/Zilliqa/zq2/assets/9307127/a6e8a18d-ac56-4024-949a-73039d9c8a3e">